Opentrons/buildroot
# board/opentrons/ot2: Simplify rootfs-overlay to reflect merged /usr

Our current system image has a ["merged `/usr`"](https://www.freedesktop.org/wiki/Software/systemd/TheCaseForTheUsrMerge/) hierarchy: `/bin`, `/sbin`, and `/lib` are symlinked to their counterparts inside `/usr`.

```shell
# file /*
/bin:        symbolic link to usr/bin
# ...
/lib:        symbolic link to usr/lib
/lib32:      symbolic link to lib
# ...
/sbin:       symbolic link to usr/sbin
# ...
```

However, currrently, in this repo, we have both `/bin` et. al. and `/usr/bin` et. al. Buildroot collapses them into `/usr` at some point during its build process.

This PR moves everything to `/usr` to eliminate the redundancy and simplify our rootfs-overlay.

* Move the file contents of `/bin` and `/lib` to `/usr/bin` and `/usr/lib`
* Delete the empty `/bin` and `/lib` directories
* Fix up relevant mentions of `/bin/foo` to `/usr/bin/foo`. This doesn't functionally matter, but it's nice to be consistent.

# Manual test plan

Before merging, I will check that:

* The OT-2 boots.
* On the live system, the files `ot-commit-machine-id`, `ot-switch-partitions`, and `ot-unused-partition` continue to be available through both `/bin` and `/usr/bin`.
* Systemd continues to successfully load the `btusbstarter` and `opentrons-commit-machine-id` services.
